presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -204,7 +204,8 @@ private boolean verifySubqueryReferenceToSourceScope(Expression referenceExpress
         @Override
```

Comment:
Now I'm almost sure it's about "Analyze SELECT..." commits :)

I'm not editing commit messages yet not to break GH's PR history. So far, all requested changes are as incremental fixups.